docs/ovn-troubleshooting.md

```diff
@@ -0,0 +1,386 @@
+- [Purpose](#purpose)
+- [Find node for an instance](#find-node-for-an-instance)
+- [Kube-OVN kubectl plugin](#kube-ovn-kubectl-plugin)
+- [List all pods on a node](#list-all-pods-on-a-node)
+- [Finding OVN and OVS pods](#finding-ovn-and-ovs-pods)
+- [Getting a shell or running commands on the OVS/OVN pods](#getting-a-shell-or-running-commands-on-the-ovsovn-pods)
+- [Run ovs-vsctl list manager](#run-ovs-vsctl-list-manager)
+- [kubectl ko vsctl <node> list manager](#kubectl-ko-vsctl-node-list-manager)
+- [Run ovs-vsctl show](#run-ovs-vsctl-show)
+- [Find the tap devices for an instance](#find-the-tap-devices-for-an-instance)
+- [Use the the MySQL command line client for the DB](#use-the-the-mysql-command-line-client-for-the-db)
+- [On Ubuntu](#on-ubuntu)
+
+
+# Purpose
+
+This page contains various troubleshooting tasks.
+
+# Find node for an instance
+
+You might want to check that the node has the required pods running, or do other
+troubleshooting tasks.
+
+This will tell you what node an instance runs on.
+
+```
+openstack server show <INSTANCE_UUID> -c hypervisor_hostname
+```
+
+# Kube-OVN kubectl plugin
+
+- Kube-OVN has a `kubectl` plugin.
+- You can see documentation at [the Kube-OVN documentation](https://kubeovn.github.io/docs/v1.12.x/en/)
+- You should install it from wherever you would like to use `kubectl` from, as described
+      [here](https://kubeovn.github.io/docs/v1.12.x/en/ops/kubectl-ko/#plugin-installation)
+    - However, you will also find it already on installed on _Genestack_ Kubernetes
+      nodes, so you can use it with `kubectl` there, if desired.
+- You can use this to run many OVS commands for a particular node
+    - if, for instance, you've found a node for a particular instances as
+      discussed above
+
+You can get help like:
+
+```
```

Comment:
Nit, all of these ``` options should have a content type, something like **``` shell** or ``` yaml**.

With that content type, we'll have a better doc experience with syntax highlighting.

Reply:
okay, pushed. Feel free to merge or request additional changes.